packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx

```diff
@@ -43,5 +47,9 @@ export const toVisTimelineOptions = (options: TimelineOptions): VisTimelineOptio
     };
   }
 
+  if (typeof options.margin !== 'undefined') {
```

Comment:
could be just `options.margin !== undefined`

Reply:
@propakov Yes we can, `typeof` approach is a more universal and recommended way to check if variables are `undefined`.